.github/workflows/build-doc.yml

```diff
@@ -147,6 +147,20 @@ jobs:
           ln -s ../../fragnova/docs/fragnova-network
           cd docs/learn
           ln -s ../../../../fragnova/docs/fragnova-network
+      - name: Step to start local HTTP server with Fragnova docs
+        run: |
+          cd shards/docs && nohup mkdocs serve > nohup.out 2> nohup.err < /dev/null &
+      - name: Wait for the database to start
+        run: wget -qO- https://raw.githubusercontent.com/eficode/wait-for/WAIT_FOR_VERSION/wait-for | sh -s -- 127.0.0.1:8000 -- echo "Docs website is up"
```

Comment:
we should fork and add to shards-lang org probably to avoid becoming broken

Reply:
I don't understand.

Reply:
`/eficode/wait-for/` repo, fork and use our fork